### PR TITLE
fix: content layout shift when image is loaded

### PR DESCRIPTION
**Pull request by cisel**
# Description
Added aspect ratio wrapper to prevent content layout shift when the image is loaded.

**Money**
**Vertical**
Screenshot 2021-06-10 at 13 23 55
Screenshot 2021-06-10 at 13 34 43

**Horizontal**
Screenshot 2021-06-10 at 13 24 36
Screenshot 2021-06-10 at 14 15 41

**Uswitch**
**Vertical**
Screenshot 2021-06-10 at 13 24 11
Screenshot 2021-06-10 at 13 34 33

**Horizontal**
Screenshot 2021-06-10 at 13 24 25
Screenshot 2021-06-10 at 13 33 53

**Bankrate**
**Vertical**
Screenshot 2021-06-10 at 13 23 39
Screenshot 2021-06-10 at 13 34 52

**Horizontal**
Screenshot 2021-06-10 at 13 24 47
Screenshot 2021-06-10 at 13 34 15

# Checklist

Pull request contains:

- [ ] A new component
- [x] Component maintenance: improvement / bug fix / etc
- [ ] Component library change: storybook / webpack / etc

Definition of done:

- [x] Includes theme changes for both Uswitch and money
- [ ] Work has been tested in multiple browsers
- [x] PR description includes description of change
- [x] PR description includes screenshot of change
- [ ] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [ ] If introducing a new component behaviour, added a story to cover that case.

### CODE REVIEW

**Review by samhstn — DISMISSED** [review]
Yep, no visual change from Percy 👍

File: src/themes/broadband-deals/theme.json [hunk 1]
```diff
@@ -870,6 +870,12 @@
           "lineHeight": "md"
         }
       },
+      "aspect-ratio-wrapper": {
+        "height": "0",
+        "paddingTop": "calc(189.67 / 336.98 * 100%)",
```
Comment (samhstn) [inline src/themes/broadband-deals/theme.json:875]:
```suggestion
        "paddingTop": "56.29%",
```

File: src/themes/bankrate/theme.json [hunk 2]
```diff
@@ -1376,6 +1376,12 @@
         },
         "mr": "base"
       },
+      "aspect-ratio-wrapper": {
+        "height": "0",
+        "paddingTop": "calc(192.69 / 342.33 * 100%)",
```
Comment (samhstn) [inline src/themes/bankrate/theme.json:1381]:
```suggestion
        "paddingTop": "56.29%",
```

File: src/themes/journey/theme.json [hunk 3]
```diff
@@ -1224,9 +1224,14 @@
       "sm": {
         "image": { "width": "100%", "height": "auto" }
       },
-
       "lg": {
         "image": { "width": 278, "height": 230 }
+      },
+      "aspect-ratio-wrapper": {
+        "height": "0",
+        "paddingTop": "calc(208.59 / 370.58 * 100%)",
```
Comment (samhstn) [inline src/themes/journey/theme.json:1232]:
```suggestion
        "paddingTop": "56.29%",
```

**Review by samhstn — COMMENTED** [review]
Where do these numbers come from?

**Review by jonathan-fielding — CHANGES_REQUESTED** [review]
looks a bit broken in percy

File: src/themes/broadband-deals/theme.json (hunk 1 above)
Comment (cisel) [reply src/themes/broadband-deals/theme.json:875]: They are image's width and height, calculating aspect-ratio. I will change them just to percentages. Thanks.

**Review by samhstn — DISMISSED** [review]
LGTM 👍

**Review by samhstn — APPROVED** [review]
LGTM 👍

**Pull request closed without merging**